### ISSUE TITLE
Will I be able to make executables with this?

### ISSUE THREAD

**Issue by vapniks** [tromey/el-compilador#1]
Is it possible to make standalone executables with this?
That would open up a whole lot of possibilities..

**Comment by tromey** [comment]
I hadn't really planned to do that but I'm not opposed to it.

**Comment by larsbrinkhoff** [comment]
In a similar-ish vein, it would be possible to have the C backend make shared objects, which could then be loaded into Emacs 25.

**Comment by tromey** [comment]
Yeah, that's totally doable.  Though I have a more radical idea, which is to use [Emacs FFI](https://github.com/tromey/emacs-ffi) to pull in [gcc-jit](https://github.com/tromey/emacs-gcc-jit) and then lower the el-compilador IR to jitted code.  Maybe going via C would be simpler, though; and the resulting shared object could be easily loaded using emacs-ffi; the main advantage being that the Emacs module API carries a considerable overhead.
